egs/librispeech/s5/local/lookahead/run_lookahead.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+. path.sh
```

Comment:
Please use ./path.sh in case . is not on PATH

Reply:
Done